README.md

```diff
@@ -1,79 +1,28 @@
-Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
-
- ```
-
-Instructions
--------
+# Take-Away Challenge
```

Comment:
Try to keep your commits small and commit a single feature at a time. It might be a good idea to try writing a test, use TDD to pass the test, and then commit (after refactoring, if needed).